Content.Shared/Traits/Assorted/IlliterateComponent.cs

```diff
@@ -0,0 +1,16 @@
+using Content.Shared.Paper;
+using Robust.Shared.GameStates;
+
+namespace Content.Shared.Traits.Assorted;
+
+/// <summary>
+/// This is used for the illiterate trait.
+/// </summary>
+[RegisterComponent, NetworkedComponent]
+public sealed partial class IlliterateComponent : Component
+{
+    /// <summary>
+    /// What message should be displayed when the character fails to write?
+    /// </summary>
+    public LocId FailWriteMessage = "paper-component-illiterate";
+}
```

Comment:
_DV and replace this unused body with ; as this isnt a datafield and there is no code using it